release/golden_notebook_tests/workloads/torch_tune_serve_test.py

```diff
@@ -180,7 +180,10 @@ def setup_serve(model, use_gpu: bool = False):
     )  # Start on every node so `predict` can hit localhost.
     serve.run(
         MnistDeployment.options(
-            num_replicas=2, ray_actor_options={"num_gpus": bool(use_gpu)}
+            num_replicas=2,
+            ray_actor_options={"num_gpus": 1, "resources": {"worker": 1}}
+            if use_gpu
+            else {},
```

Comment:
this won't spread across nodes if no GPU is used, is that intentional?

Reply:
Yeah I think `no_gpu` is used for smoke test. There is only one worker node type in this test, so no gpu should mean all replicas get started on head node.

Reply:
ok